analyzer/src/funTest/assets/projects/synthetic/spdx-project-expected-output.yml

```diff
@@ -54,8 +54,8 @@ packages:
     url: "<REPLACE_URL_PROCESSED>"
     revision: "<REPLACE_REVISION>"
     path: "analyzer/src/funTest/assets/projects/synthetic/spdx/project/libs/curl"
-- id: "SpdxDocumentFile::openssl:1.1.1g"
-  purl: "pkg:spdxdocumentfile/openssl@1.1.1g"
+- id: "SpdxDocumentFile:OpenSSL Development Team:openssl:1.1.1g"
+  purl: "pkg:spdxdocumentfile/OpenSSL%20Development%20Team/openssl@1.1.1g"
```

Comment:
You are changing the id & purl in this commit but there no arguments in this commit for the reason why.

Reply:
Explanation was added :)